app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -435,21 +446,78 @@ export default {
         this.showMentions = false;
       }
     },
+    replyType(updatedReplyType, oldReplyType) {
+      this.setToDraft(this.conversationId, oldReplyType);
+      this.getFromDraft();
+    },
   },
 
   mounted() {
+    this.getFromDraft();
     // Donot use the keyboard listener mixin here as the events here are supposed to be
     // working even if input/textarea is focussed.
     document.addEventListener('keydown', this.handleKeyEvents);
     document.addEventListener('paste', this.onPaste);
 
+    this.draftAutoSaveInterval = setInterval(() => {
+      this.saveDraft();
+    }, 10000);
```

Comment:
Instead of using an interval, we can use the debounce concept which happens as we stop typing. Example below
```js
saveDraft(e) {
      this.message = e.target.value;
      clearTimeout(this.debounce);
      this.debounce = setTimeout(() => {
             // save the draft here
      }, 600);
    },
```

and add `@input="saveDraft"` to the input box

Reply:
This sounds good. Let try to do this @nithindavid.